### PR TITLE
Forms Automated Test - LPS-131245 | master

### PR DESCRIPTION
**Pull request by c3saroliveira**

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsParagraphField.testcase [hunk 1]
```diff
@@ -195,12 +195,47 @@ definition {
...
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "1",
+			fieldType = "Paragraph");
+
+		var fieldParagraphField = FormFields.getFieldName();
```
Comment (CalumR23) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsParagraphField.testcase:214]: You are declaring a var here but it is not being referenced elsewhere. Is it possible to take this line out and have the test still pass?

**Comment by CalumR23** [comment]
ci:test:relevant

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsParagraphField.testcase [hunk 2]
```diff
@@ -195,12 +195,57 @@ definition {
 	}
 
 	@description = "Verify that inputs to the Body Text can be undone and redone"
-	@ignore = "true"
 	@priority = "5"
 	test CanUndoAndRedo {
 		property portal.acceptance = "false";
```
Comment (natocesarrego) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsParagraphField.testcase:200]: Can you please add `property forms.functional = "true";` here following alphabetic order?

I mean, something like:
```
property forms.functional = "true";
property portal.acceptance = "false";
```

Thanks.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsParagraphField.testcase [hunk 3]
```diff
@@ -195,12 +195,49 @@ definition {
...
-		// TODO LPS-131245 CanUndoAndRedo pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
```
Comment (brittneyq) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsParagraphField.testcase:210]: For line 210:

Please refer to https://github.com/liferay-forms/liferay-portal/pull/233#discussion_r632481834 

Should change `fieldPositionNumber` to 0.

**Comment by brittneyq** [comment]
ci:test:sf

**Comment by diogocordeiro** [comment]
ci:forward

**Pull request closed without merging**